guides/tutorials/codelabs/pubsub-to-cf/index.md

```diff
@@ -102,31 +104,29 @@ e. Apply your changes:
 
 ## 5. Configure the Application and Pipeline
 
-a. Create a new pipeline for your application. In the pipeline configuration, configure the Expected Artifacts as shown below, a GCS object with the path to the application archive in your GCS bucket:
+a. Create a new pipeline for your application. In the pipeline configuration, under Automated Triggers, add a new trigger of **Type** "Pub/Sub". Select GCP Cloud Pub/Sub as the **Pub/Sub System Type**, select your GCP Cloud Pub/Sub Spinnaker subscription in **Subscription Name**, and under **Attribute Constraints**, add an entry with the key `eventType` and value `OBJECT_FINALIZE ` (see the [Google Cloud Storage documentation](https://cloud.google.com/storage/docs/pubsub-notifications)).
 
   {% include figure
-     image_path="./expected-artifacts.png"
+     image_path="./add-a-trigger.png"
   %}
 
-b. Configure Automated Triggers, adding the GCP Cloud Pub/Sub Spinnaker subscription as a trigger and selecting the Artifact Constraints:
+b. In the **Artifact Constraints** dropdown, select "Define a new artifact..." to bring up the **Expected Artifact** form. Enter a **Display Name** or keep the auto-generated default. Select your GCS account in the **Account** dropdown and enter the path to the artifact in the **Object path** field.
 
   {% include figure
-     image_path="./triggers.png"
+     image_path="./expected-artifact.png"
   %}
 
-c. Add a Deploy stage to the pipeline. Create a new server group and provide details on deployment settings, the application artifact, and the manifest artifact:
+Click **Save Artifact**.
```

Comment:
This should get a number (letter). In fact, I think you can replace all the letters with with `a` in each case, and it should order them as it does with numbers.

Reply:
I seemed to recall having tried that and not being able to get it to work, but I did add a letter. Thank you!